firmware/gameport-adapter/Logitech.h

```diff
@@ -296,6 +332,11 @@ class Logitech : public Joystick {
       m_metaData.deviceName[i] = getBits(packet, 66 + 8 * i, 8);
     }
 
+    m_metaData.isThunder = 0;
+    if(strcmp(m_metaData.deviceName, "ThunderPad") == 0){
+      m_metaData.isThunder = 1;
+    }
+
```

Comment:
This can be removed completely, since the information is already encoded in the deviceID